extra-dev/packages/coq-elpi/coq-elpi.dev/opam

```diff
@@ -1,38 +1,44 @@
+# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
-authors: [ "Enrico Tassi" ]
+synopsis: "Elpi extension language for Coq"
+description:
+  "Coq-elpi provides a Coq plugin that embeds ELPI. It also provides a way to embed Coq's terms into λProlog using the Higher-Order Abstract Syntax approach and a way to read terms back. In addition to that it exports to ELPI a set of Coq's primitives, e.g. printing a message, accessing the environment of theorems and data types, defining a new constant and so on. For convenience it also provides a quotation and anti-quotation for Coq's syntax in λProlog. E.g., `{{nat}}` is expanded to the type name of natural numbers, or `{{A -> B}}` to the representation of a product by unfolding the `->` notation. Finally it provides a way to define new vernacular commands and new tactics."
+maintainer: ["Enrico Tassi <enrico.tassi@inria.fr>"]
+authors: ["Enrico Tassi <enrico.tassi@inria.fr>"]
 license: "LGPL-2.1-or-later"
+tags: [
+  "category:Miscellaneous/Coq Extensions"
+  "keyword:λProlog"
+  "keyword:higher order abstract syntax"
+  "logpath:elpi"
+]
 homepage: "https://github.com/LPCIC/coq-elpi"
 bug-reports: "https://github.com/LPCIC/coq-elpi/issues"
-dev-repo: "git+https://github.com/LPCIC/coq-elpi"
-
 depends: [
-  "ocaml" {>= "4.10.0"}
+  "dune" {>= "3.13"}
+  "ocaml" {>= "4.09.0"}
   "stdlib-shims"
-  "elpi" {= "1.19.2" }
-  "coq" {= "dev"}
+  "elpi" {>= "1.18.2" & < "1.20.0~"}
+  "coq" {= dev}
```

Comment:
shouldn't this be `  "coq" {>= "8.19"}` like https://github.com/LPCIC/coq-elpi/blob/master/coq-elpi.opam ?

If yes, that would be enough given `"dev" >= "8.19"`

Reply:
sincerly I don't know what is best... I'm fine with any solution that works ;-)